src/import/FSHImporter.ts

```diff
@@ -1121,6 +1116,17 @@ export class FSHImporter extends FSHVisitor {
     }
   }
 
+  visitInvariantRule(ctx: pc.InvariantRuleContext): AssignmentRule | InsertRule {
+    if (ctx.fixedValueRule()) {
+      return this.visitFixedValueRule(ctx.fixedValueRule());
+    } else if (ctx.insertRule()) {
+      return this.visitInsertRule(ctx.insertRule());
+    } else if (ctx.pathRule()) {
+      this.visitPathRule(ctx.pathRule(), true);
```

Comment:
Oh. Hmmm... I guess I should check what that boolean actually does, but maybe not.

Reply:
I think it's used for correctly updating soft indexes on path rules, and I think since the rules are tossed out now, the soft index calculations should work like they do for non-Instance entity rules. This is around line 2175 - 2179.

Reply:
Thanks! I doubt we even have that test for all the other entity types, so nifty bonus!